### PR TITLE
doc(connect): init method

### PR DESCRIPTION
**Pull request by mroz22**
cc @Hannsek

### CODE REVIEW

File: docs/packages/connect/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ TrezorConnect.init({
 });
 ```
 
-Starting with Trezor Connect 7, we have implemented a new feature — Trezor Connect Manifest — which requires that you as a Trezor Connect integrator, to share your e-mail and application url with us.
+Trezor Connect Manifest requires that you as a Trezor Connect integrator share your e-mail and application url with us.
```
Comment (Hannsek) [inline docs/packages/connect/index.md]:
```suggestion
Trezor Connect Manifest requires that you, as a Trezor Connect integrator, share your e-mail and application url with us.
```

**Review by Hannsek — CHANGES_REQUESTED** [review]

**Comment by sime** [comment]
This might close https://github.com/trezor/trezor-suite/issues/7158

**Comment by Hannsek** [comment]
No, it will not. It is completely different thing.

File: docs/packages/connect/methods/init.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+```
+
+### Params
+
+[init type](https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/types/api/init.ts)
+
+-   `manifest` - _required_ `Object` of type `[Manifest]`(https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/types/settings.ts#L3),
+-   `connectSrc` - _optional_ `string`. Set custom src for TrezorConnect iframe.
+-   `debug` - _optional_ `boolean`, default `false`. Prints debug logs.
+-   `transportReconnect` - _optional_ `boolean`, default `true`. If transport dies during lifecycle if application, this field decides whether TrezorConnect tries to reestablish connection with transport layer.
```
Comment (humanumbrella) [inline docs/packages/connect/methods/init.md]:
```suggestion
-   `transportReconnect` - _optional_ `boolean`, default `true`. If transport dies during lifecycle of application, this field decides whether TrezorConnect tries to reestablish connection with transport layer.
```

File: docs/packages/connect/methods/init.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+-   `pendingTransportEvent` - _optional_ `boolean`, default `true`. Postpone emitting TRANSPORT.START event to the moment when connected device becomes available.
+-   `interactionTimeout` - _optional_ `number`, default `600`. Time in milliseconds after which popup automatically closes.
```
Comment (humanumbrella) [inline docs/packages/connect/methods/init.md]: i think this may be seconds not milliseconds

Comment (mroz22) [reply docs/packages/connect/methods/init.md]: you are right, my bad, will fix

**Comment by mroz22** [comment]
applied suggested changes and forcepushed

**Review by Hannsek — APPROVED** [review]

**Pull request merged**